### PR TITLE
 Multi-GPU - Add information- how to export AQ_DRM_DEVICES for uwsm users

### PR DESCRIPTION
**Pull request by izmyname**
See https://github.com/Vladimir-csp/uwsm/pull/62#issuecomment-2479742978

@Vladimir-csp pinging for corrections

### CODE REVIEW

File: pages/Configuring/Multi-GPU.md [hunk 1]
```diff
@@ -73,3 +73,14 @@ whatever reason, use `card1`.
 
 You should now be able to use an integrated GPU for lighter GPU loads,
 including Hyprland, or default to your dGPU if you prefer.
+
+{{< callout type=info >}}
+
+[uwsm](../../Useful-Utilities/Systemd-start) users are adiviced to export `AQ_DRM_DEVICES` variable inside `~/.config/uwsm/env-hyprland`, instead.
```
Comment (fufexan) [inline pages/Configuring/Multi-GPU.md:79]:
```suggestion
[uwsm](../../Useful-Utilities/Systemd-start) users are advised to export the `AQ_DRM_DEVICES` variable inside `~/.config/uwsm/env-hyprland`, instead. 
```

**Review by fufexan — APPROVED** [review]

**Comment by izmyname** [comment]
**UPD:** I just noticed - Hyprland-wiki website hasn't been updated.  Maybe, something bugged?

**Comment by fufexan** [comment]
@vaxerski? The changes have propagated to the backend but maybe not to the server.

**Comment by vaxerski** [comment]
fixed. I really need to migrate the wiki from the current host. The watcher dies every now and then

**Pull request merged**